tests/compile-fail/inhabited_type.rs

```diff
@@ -0,0 +1,7 @@
+ // error-pattern: the evaluated program aborted
```

Comment:
The test name is odd. What about "uninit_uninhabited_type"?

Also, please check here that the "attempted to instantiate uninhabited type" is part of the message.

Reply:
you're right. was weird for me too, choose it because the intrinsic is called `assert_inhabited` so I choose inhabited even though uninhabited/uninhabitable sounds better. will change.